jobs/integr8ly/poc-testing-executor.yaml

```diff
@@ -42,8 +42,9 @@ This needs to be changed to 'false' for Integreatly Workshop since it uses valid
             default: master
             description: "Branch of a repository of Integreatly test suites"
         - string:
-            name: CLUSTER_URL
-            description: "URL of cluster for installation."
+            name: YOURCITY
+            default: integreatly-qe.opentry.me
+            description: "City or Customer (5 char min.) plus the generated hash, e.g. qebrno-5d10 [required]"
```

Comment:
The description here is not correct. Also, it should be changed if you agree with my previous comment. My proposal would be "Hostname of the cluster. Typically it is just a cluster URL without the protocol part [required]"